src/command/player/seasonStatsCompare.js

```diff
@@ -0,0 +1,232 @@
+      teamMainColor ? teamMainColor.hex : '#000'
+    }') ${teamAbbreviation}} {bold.white #${jersey} ${displayFirstLast} │ ${
+      seasonTtpe
+    }}`;
+    nameStr += `${playerName}\n`;
+  });
+  return nameStr;
+};
+
+// make an object where keys are the seasons
+// each season maps to an array of the player stats for that season
+// each array has the length of the number of players, entries initalized to empty object
+const makeSeasonObj = (playerProfile, seasonStr) => {
+  const seasonObj = {};
+  let index = 0;
+  playerProfile.forEach(player => {
+    // access the season totals for either post or regular season
+    const seasonArr = player[`seasonTotals${seasonStr}`];
+    seasonArr.forEach(season => {
+      const currentSeason = season.seasonId;
+      // check if the season is already in the object
+      if (seasonObj[currentSeason]) {
+        // if it does add this players data to the array
+        seasonObj[currentSeason][index] = season;
+      } else {
+        // otherwise create a new array with length of number of players
+        seasonObj[currentSeason] = [...Array(playerProfile.length)].fill({});
+        seasonObj[currentSeason][index] = season;
+      }
+    });
+    index += 1;
+  });
+  return seasonObj;
+};
+
+// takes in all the data and returns an array with the player's totals as each entry
+const makeOverall = (playerProfile, seasonStr) => {
+  const overallArr = [];
+  playerProfile.forEach(player => {
+    overallArr.push(...player[`careerTotals${seasonStr}`]);
+  });
+  return overallArr;
+};
+
+// takes in the data for a season and combines the stats into strings to be pushed to table
+/* eslint-disable no-param-reassign */
+const makeRow = seasonData => {
+  const template = {
+    teamAbbreviation: [],
+    playerAge: [],
+    gp: [],
+    min: [],
+    pts: [],
+    fgPct: [],
+    fg3Pct: [],
+    ftPct: [],
+    ast: [],
+    reb: [],
+    stl: [],
+    blk: [],
+    tov: [],
+  };
+  let seasonId;
+  seasonData.forEach(player => {
+    // configure certain data before we append to string
+    // make sure object isn't empty
+    if (Object.keys(player).length !== 0) {
+      player.fg3Pct = (player.fg3Pct * 100).toFixed(1);
+      player.fgPct = (player.fgPct * 100).toFixed(1);
+      player.ftPct = (player.ftPct * 100).toFixed(1);
+      // overall stats dont have team abbreviation, have to check if exists
+      if (player.teamAbbreviation) {
+        const teamMainColor = getMainColor(player.teamAbbreviation);
+        player.teamAbbreviation = chalk`{bold.white.bgHex('${
+          teamMainColor ? teamMainColor.hex : '#000'
+        }') ${player.teamAbbreviation}}`;
+      }
+      if (!template.seasonId) {
+        seasonId = bold(player.seasonId);
+      }
+    }
+    Object.keys(template).forEach(key => {
+      // add data to str or if no data put in a dash
+      template[key].push(player[key] || '-');
+    });
+  });
+
+  Object.keys(template).forEach(key => {
```

Comment:
This project has installed `Ramda`, so maybe this line could be replaced by [R.forEachObjIndexed](http://ramdajs.com/docs/#forEachObjIndexed)